cli/pcluster/utils.py

```diff
@@ -93,7 +93,9 @@ def get_region():
 
 def get_partition():
     """Get partition for the AWS_DEFAULT_REGION set in the environment."""
-    return "aws-us-gov" if get_region().startswith("us-gov") else "aws"
+    region = get_region()
+    partition = [partition for partition in ["us-gov", "cn"] if region.startswith(partition + "-")]
```

Comment:
I think there is still a problem because we need to prepend aws to the partition: `aws-us-gov`, `aws-cn`

quick tip: you could simplify by using next with something like:
```
return next(("aws" + partition for partition in ["us-gov", "cn"] if region.startswith(partition)), "aws")
```

Reply:
cool one-liner :)
Just very minor fix: 
```
return next(("aws-" + ...
```